docs/legacy/porting.md

```diff
@@ -23,3 +24,4 @@ Here you can find a list of primers on how to port from old versions to the curr
 [1194to120]: https://gist.github.com/ChampionAsh5357/cf818acc53ffea6f4387fe28c2977d56
 [1204to1205]: https://gist.github.com/ChampionAsh5357/53b04132e292aa12638d339abfabf955
 [1206to121]: https://gist.github.com/ChampionAsh5357/d895a7b1a34341e19c80870720f9880f
```

Comment:
We have primers down to 1.19.3 on our github repo. The repo link should be used here:
https://github.com/neoforged/.github/tree/main/primers

Reply:
This I will leave for a separate PR since I want to update all the links at once.